### PR TITLE
add data-in doc for cloud

### PR DESCRIPTION
**Pull request by wingwing2005**

### CODE REVIEW

File: docs/en/04-data-in/01-ds/10-opentsdb.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ OpenTSDB data source is the process of writing data from the OpenTSDB server to
 ## Prerequisites
 
 - Create an empty database to store your OpenTSDB data. For more information, see [Database](../../../programming/model/#create-database).
-- Ensure that the connection agent is running on a machine located on the same network as your OPC Server. For more information, see [Install Connection Agent](../install-agent/).
+- (Optional) Ensure that the connection agent is running on a machine located on the same network as your OPC Server. For more information, see [Install Connection Agent](../install-agent/).
```
Comment (danielclow) [inline docs/en/04-data-in/01-ds/10-opentsdb.md]: "on the same network as your OpenTSDB instance"?

File: docs/en/04-data-in/01-ds/13-mysql.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+- Create an empty database to store your MySQL data. For more information, see [Database](../../../programming/model/#create-database).
+- (Optional) Ensure that the connection agent is running on a machine located on the same network as your MySQL Server. For more information, see [Install Connection Agent](../install-agent/).
+
+## Procedure
+
+1. In TDengine Cloud, open **Data In** page. On the **Data Sources** tab, click **Add Data Source** button to open the new data source page.
+   - In the **Name** field, fill in the name of the data source.
+   - In the **Type** field, select the type of **MySQL**.
+   - In the **Agent** selection which is not not a mandatory field, select the agent you have already created, or if you have not created a agent, click the **Create New Agent** button to create it.
```
Comment (danielclow) [inline docs/en/04-data-in/01-ds/13-mysql.md]: - (Optional) In the **Agent** drop-down list, select the agent ...

File: docs/en/04-data-in/01-ds/14-postgres.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- Create an empty database to store your PostgreSQL data. For more information, see [Database](../../../programming/model/#create-database).
+- (Optional) Ensure that the connection agent is running on a machine located on the same network as your PostgreSQL Server. For more information, see [Install Connection Agent](../install-agent/).
+
+## Procedure
+
+1. In TDengine Cloud, open **Data In** page. On the **Data Sources** tab, click **Add Data Source** button to open the new data source page.
+   - In the **Name** field, fill in the name of the data source.
+   - In the **Type** field, select the type of **PostgreSQL**.
+   - In the **Agent** selection which is not not a mandatory field, select the agent you have already created, or if you have not created a agent, click the **Create New Agent** button to create it.
```
Comment (danielclow) [inline docs/en/04-data-in/01-ds/14-postgres.md]: same as mysql

File: docs/en/04-data-in/01-ds/15-oracle.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- Create an empty database to store your Oracle data. For more information, see [Database](../../../programming/model/#create-database).
+- (Optional) Ensure that the connection agent is running on a machine located on the same network as your Oracle Server. For more information, see [Install Connection Agent](../install-agent/).
+
+## Procedure
+
+1. In TDengine Cloud, open **Data In** page. On the **Data Sources** tab, click **Add Data Source** button to open the new data source page.
+   - In the **Name** field, fill in the name of the data source.
+   - In the **Type** field, select the type of **Oracle**.
+   - In the **Agent** selection which is not not a mandatory field, select the agent you have already created, or if you have not created a agent, click the **Create New Agent** button to create it.
```
Comment (danielclow) [inline docs/en/04-data-in/01-ds/15-oracle.md]: same as mysql

File: docs/en/04-data-in/01-ds/10-opentsdb.md (hunk 1 above)
Comment (wingwing2005) [reply docs/en/04-data-in/01-ds/10-opentsdb.md]: Yes.

**Comment by wingwing2005** [comment]
@danielclow  please review again, thanks.

File: docs/en/04-data-in/01-ds/10-opentsdb.md (hunk 1 above)
Comment (danielclow) [reply docs/en/04-data-in/01-ds/10-opentsdb.md]: I mean, the current text is "Ensure that the connection agent is running on a machine located on the same network as your OPC Server." I guess it should be "Ensure that the connection agent is running on a machine located on the same network as your OpenTSDB instance." Is that correct?

**Pull request merged**